x-pack/test/api_integration/deployment_agnostic/default_configs/stateful.config.base.ts

```diff
@@ -85,6 +89,10 @@ export function createStatefulTestConfig<T extends DeploymentAgnosticCommonServi
           `xpack.security.authc.realms.saml.${MOCK_IDP_REALM_NAME}.attributes.name=${MOCK_IDP_ATTRIBUTE_NAME}`,
           `xpack.security.authc.realms.saml.${MOCK_IDP_REALM_NAME}.attributes.mail=${MOCK_IDP_ATTRIBUTE_EMAIL}`,
         ],
+        files: [
+          // SAML realm roles for stateful
```

Comment:
__nit:__ Maybe it would be clearer if instead we say that we're shimming the roles that are equivalent to the ones we have in serverless or something along these lines. SAML is really an implementation detail in this context.